specs/trip_dispatch_spec.rb

```diff
@@ -89,4 +89,90 @@
       passenger.trips.must_include trip
     end
   end
+
+  describe "request_trip" do
+    before do
+
+      @trip_data = {
+        id: 601,
+        driver: RideShare::Driver.new(id: 3, name: "Lovelace", vin: "12345678912345678"),
+        passenger: RideShare::Passenger.new(id: 1, name: "Ada", phone: "412-432-7640"),
+        start_time: Time.now,
+        end_time: nil,
+        cost: nil,
+        rating: nil
+      }
+      @trip = RideShare::Trip.new(@trip_data)
+    end
+
+    it "Create a new instance of Trip" do
+      @trip.must_be_kind_of RideShare::Trip
```

Comment:
This test doesn't run any of `TripDispatcher`'s methods. Is it in the wrong place?